docs/outlook/sideload-outlook-add-ins-for-testing.md

```diff
@@ -167,7 +79,7 @@ To manually sideload an add-in in the new Outlook on Windows (preview), see [Sid
 
     ![The Get Add-ins button is selected in Outlook on Mac.](../images/outlook-sideload-mac-classic.png)
 
-    > [!IMPORTANT]
+    > [!TIP]
     > If you don't see **Get Add-ins** in your version of Outlook, select the ellipsis button (`...`) from the ribbon, then select **Get Add-ins**.
     >
     > ![The Get Add-ins option is selected from the ellipsis button in Outlook on Mac.](../images/outlook-sideload-new-mac.png)
```

Comment:
The image in step 4 (and the other tab's step 3) says the dialog title is "Custom Addins". We should update to match. Also, do they plan to fix that in the UI?

Reply:
Fixed. I'm not sure if it's on their backlog to get it fixed. I can follow up.